### DISCUSSION TITLE
backup source relative path?

### DISCUSSION THREAD

**Discussion by IndoMK** [garethgeorge/backrest#538] [Q&A] [answered]
Question. And apologies if I missed this in the documentation anywhere, I swear I read through it twice before asking. 

I originally started out with vanilla restic using cd to backup source and relative paths as discussed under the "Skip creating snapshots if unchanged" section in the restic docs. 

> Note that when using absolute paths to specify the backup source, then also changes to the parent folders result in a changed snapshot. For example, a backup of /home/user/work will create a new snapshot if the metadata of either /, /home or /home/user change. To avoid this problem run restic from the corresponding folder and use relative paths.

My question is does Backrest support relative paths in this same way? And if yes, how can I configure it in my backup plan?

**Comment by garethgeorge** [comment, marked as answer]
Relative paths are “supported,” but probably aren’t best practice. The paths will be relative to the working directory of the backrest binary. In docker this may not stay the same. Using a bare install you can control the working directory— but explicit paths are always safest.

**Reply by IndoMK** [reply]
Thank you for the insight! I'll stay with absolute paths as I roll out Backrest across my machines.
